systemtest/src/test/java/io/strimzi/systemtest/mirrormaker/MirrorMaker2IsolatedST.java

```diff
@@ -529,7 +534,14 @@ void testMirrorMaker2CorrectlyMirrorsHeaders(ExtensionContext extensionContext)
         // Deploy Topic for example clients
         resourceManager.createResource(extensionContext, KafkaTopicTemplates.topic(kafkaClusterSourceName, testStorage.getTopicName()).build());
 
-        resourceManager.createResource(extensionContext, KafkaMirrorMaker2Templates.kafkaMirrorMaker2(testStorage.getClusterName(), kafkaClusterTargetName, kafkaClusterSourceName, 1, false).build());
+        resourceManager.createResource(extensionContext, KafkaMirrorMaker2Templates.kafkaMirrorMaker2(testStorage.getClusterName(), kafkaClusterTargetName, kafkaClusterSourceName, 1, false)
+                .editSpec()
+                    .editMirror(0)
+                        .editSourceConnector()
+                            .addToConfig("refresh.topics.interval.seconds", "1")
```

Comment:
Should this config be a part of the "default" MM2 resource in the `KafkaMirrorMaker2Templates` rather than adding it to every test?

Reply:
yes I was thinking about it but I didn't want to introduce it in this PR